### ISSUE TITLE
support pdf preview

### ISSUE THREAD

**Issue by akosyakov** [yandeu/five-server-vscode#1]
currently it just shows the content

motivated by https://github.com/gitpod-io/gitpod/issues/3831#issuecomment-824030132

**Comment by yandeu** [comment]
Just played around with it a little bit. 

Is this what you are looking for?

*left: before - right: after*

<img width="1175" alt="pdf-preview" src="https://user-images.githubusercontent.com/20306025/115564842-25195a00-a2b9-11eb-9c29-b615ab339c18.png">

**Comment by yandeu** [comment]
I have added this preview in https://github.com/yandeu/five-server/commit/58d63fe268fec976882615135d086bd2b9896041

Btw, you could also write your own middleware. The example below works with the current version of Five Server and will show the PDF file in full screen.

---

```js
// fiveserver.config.js
const pdfMiddleware = (req, res, next) => {
  if (/\.pdf\.preview/.test(req.url)) {
    const html = `
        <!DOCTYPE html>
        <html>
          <body>
            <style>
              html,
              body {
                margin: 0;
                padding: 0;
                overflow: hidden;
              }
            </style>
            <iframe 
              style="width: 100vw; height: 100vh;"
              frameborder="0" 
              scrolling="no"                
              width="100%" height="100%"
              src="${req.url.replace('.preview', '')}">
          </iframe>
          </body>
        </html>
      `
    return res.type('html').send(html)
  } else return next()
}

module.exports = {
  middleware: [pdfMiddleware]
}
```

**Comment by yandeu** [comment]
The PDF Preview support is now available in version >0.0.28
